src/Components/Explore/Explore.css

```diff
@@ -0,0 +1,20 @@
+.Explore {
```

Comment:
Avoid using capital letters for className.

Reply:
okay changed it.